package.json

```diff
@@ -24,7 +24,7 @@
     "lint": "tslint src",
     "coverage": "jest --config .jestrc --coverage",
     "release": "cd dist && npm publish",
-    "build": "./scripts/build"
+    "build": "node ./scripts/build.js"
```

Comment:
Keep `build` tied to the bash script, and create a `build:windows`, that way we can compare during the review

Reply:
Ok, will do.